src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+body,
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --header-heigth: 60px;
```

Comment:
There is a typo in the CSS variable name `--header-heigth`. It should be corrected to `--header-height` to ensure consistency and avoid potential issues in the styles that rely on this variable.